lib/activesupport/all/activesupport.rbi

```diff
@@ -1074,43 +1343,54 @@ end
 # defines some of the methods at https://github.com/rails/rails/tree/v6.0.0/activesupport/lib/active_support/core_ext/hash
 # this is not a complete definition!
 class Hash
-  sig { returns(T::Hash[String, T.untyped]) }
+  sig { params(other_hash: T::Hash[T.untyped, T.untyped]).returns(T::Hash[T.untyped, T.untyped]) }
+  def deep_merge(other_hash, &block); end
+
+  sig { params(keys: T.untyped).returns(T.self_type) }
+  def except(*keys); end
+
+  sig { returns(T::Hash[String, V]) }
```

Comment:
`symbolize_keys` and `stringify_keys` only operate on the keys so whatever the `V` is should be unchanged.